File: olivier/projetFormation/src/main/java/projetFormation/model/Adresse.java
package projetFormation.model;

import javax.persistence.Embeddable;

@Embeddable
public class Adresse {

	/**
	 * Default constructor
	 */
	public Adresse() {
	}

	/**
	 * 
	 */
	private int numero;

	/**
	 * 
	 */
	private String rue;

	/**
	 * 
	 */
	private String codePostal;

	/**
	 * 
	 */
	private String ville;

	public Adresse(int numero, String rue, String codePostal, String ville) {
		super();
		this.numero = numero;
		this.rue = rue;
		this.codePostal = codePostal;
		this.ville = ville;
	}

	public int getNumero() {
		return numero;
	}

	public void setNumero(int numero) {
		this.numero = numero;
	}

	public String getRue() {
		return rue;
	}

	public void setRue(String rue) {
		this.rue = rue;
	}

	public String getCodePostal() {
		return codePostal;
	}

	public void setCodePostal(String codePostal) {
		this.codePostal = codePostal;
	}

	public String getVille() {
		return ville;
	}

	public void setVille(String ville) {
		this.ville = ville;
	}

}

File: olivier/angular/src/app/model/adresse.ts
export class Adresse {
  private _numero: string | undefined;
  private _rue: string | undefined;
  private _codePostal: string | undefined;
  private _ville: string | undefined;

  constructor(
    numero?: string,
    rue?: string,
    codePostal?: string,
    ville?: string
  ) {
    this._numero = numero;
    this._rue = rue;
    this._codePostal = codePostal;
    this._ville = ville;
  }

  /**
   * Getter numero
   * @return {string }
   */
  public get numero(): string | undefined {
    return this._numero;
  }

  /**
   * Getter rue
   * @return {string }
   */
  public get rue(): string | undefined {
    return this._rue;
  }

  /**
   * Getter codePostal
   * @return {string }
   */
  public get codePostal(): string | undefined {
    return this._codePostal;
  }

  /**
   * Getter ville
   * @return {string }
   */
  public get ville(): string | undefined {
    return this._ville;
  }

  /**
   * Setter numero
   * @param {string } value
   */
  public set numero(value: string | undefined) {
    this._numero = value;
  }

  /**
   * Setter rue
   * @param {string } value
   */
  public set rue(value: string | undefined) {
    this._rue = value;
  }

  /**
   * Setter codePostal
   * @param {string } value
   */
  public set codePostal(value: string | undefined) {
    this._codePostal = value;
  }

  /**
   * Setter ville
   * @param {string } value
   */
  public set ville(value: string | undefined) {
    this._ville = value;
  }
}
